integration/tests/update/update_cluster_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/gexec"
+	clusterutils "github.com/weaveworks/eksctl/integration/utilities/cluster"
```

Comment:
Nit: this should be with the internal imports group

Reply:
https://github.com/eksctl-io/eksctl/pull/7117/commits/25e93e11a6a40b08f053fc0df882ff81e1338d5f